atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -176,7 +176,7 @@ def set_koji_task_annotations_whitelist(self, annotations):
         koji's configuration to be included in the build_annotations.json file,
         which will be attached in the koji task output.
         """
-        koji_config = get_koji(self.workflow, {})
+        koji_config = get_koji(self.workflow)
```

Comment:
This looks like a change in behavior. Should this be wrapped in `try/except` block?

Reply:
Even if we delete empty dict, it won't change the behavior of this function, because of 'koji' will always be in reactor_config_map('koji' is required in the schema).